File: javascriptp/switch_statement/book.js
var Book = function () {
  this.condition = "a";
  this.method = function () {
    switch (this.condition) {
      case "a":
        console.log("Case Rendered");
        break;
      default:
        console.log("Default Rendered");
    }
  };
};

const newBook = new Book();
newBook.method();
newBook.condition = "b";
newBook.method();

File: javap/switch_statement/book.java
package javap.switch_statement;

class Book {
    public char data = 'a';

    void method() {
        switch (data) {
            case 'a':
                System.out.println("Case rendered");
                break;
            default:
                System.out.println("Default rendered");
        }
    }

    public static void main(String args[]) {
        Book b1 = new Book();
        b1.method();
        b1.data = 'b';
        b1.method();
    }
}